services/masterdata/src/lib/core/services/index.ts

```diff
@@ -1,4 +1,4 @@
 export * from "./logger-messages.service";
 export * from "./masterdata-crud-http.service";
-export * from "./error-handler.service";
+export * from "../../../../../../../../apps/masterdata/src/app/error-handler.service";
```

Comment:
@dageorgero You cannot load the error handler service from the app in the framework. The other way around could work :)

Reply:
good point. it's actually i line i forgot to remove, since the time i had a ErrorHandlerService in here, not in apps/masterdata!